package.json

```diff
@@ -180,6 +180,11 @@
           "type": "string",
           "default": "",
           "description": "The path to (temporarily) store test result files in"
+        },
+        "dotnet-test-explorer.suppressedMessages": {
```

Comment:
Would better to use https://code.visualstudio.com/docs/extensionAPI/vscode-api#Memento api (refer to https://github.com/formulahendry/vscode-mysql/blob/177396b240b9760963e7447f84becf4bd1d891b0/src/mysqlTreeDataProvider.ts#L74) Since this kind of config is designed for user to update in `User Setting`, while in our case, we do not want to expose it to end user.

Reply:
Done. Though, I don't think there is an easy way to reset these now.